docs/source/explanations/security.md

```diff
@@ -238,39 +238,29 @@ pip install httpx
 3. It is recommended to set the Client Secret as an environment variable, such
    as `OIDC_CLIENT_SECRET`, and reference that from configuration file as shown
    below.
-4. Obtain the OIDC provider's public key(s). These are published by the OIDC provider.
-   Starting from a URL like:
+4. Get the OIDC provider's well-known endpoint. These are expected shared configuration values published by the OIDC provider.
```

Comment:
It's probably worth including a link to the docs for running a local OIDC provider--for testing.

[not necessarily on this line]